README.md

```diff
@@ -16,6 +16,67 @@ this is a Calculator using react js
 - Twitter: [@yahyaELganayni](https://twitter.com/@crazy20046)
 - LinkedIn: [yahyaELganayni](https://www.linkedin.com/in/yahya-el-ganayni-a456115b/)
 
+### 1. Setup
+
+- Set up Git flow
+- Set up React
+- Set up linters
+- Configure deploy
+
```

Comment:
- [ ] As previously requested, Please describe in a readme file how can a user clone your project.
- Include clear instructions to follow to follow and the necessary commands to run.